[2020-06-01 20:18:31,871] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.cvd 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-01 20:18:31,879] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.cvd 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-01 20:18:31,880] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:18:31,880] {taskinstance.py:867} INFO - Starting attempt 2 of 3
[2020-06-01 20:18:31,880] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:18:31,896] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): cvd> on 2020-05-30T00:00:00+00:00
[2020-06-01 20:18:31,898] {standard_task_runner.py:53} INFO - Started process 14709 to run task
[2020-06-01 20:18:31,952] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test_dag_cvd.cvd 2020-05-30T00:00:00+00:00 [running]> nineleaps
[2020-06-01 20:18:32,414] {taskinstance.py:1128} ERROR - 'newline' is an invalid keyword argument for this function
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/airflow/models/taskinstance.py", line 966, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python2.7/dist-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python2.7/dist-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/nineleaps/airflow/dags/airflow_test.py", line 27, in cvd_api
    cvd = open('data_covid.csv', 'w', newline='\n')
TypeError: 'newline' is an invalid keyword argument for this function
[2020-06-01 20:18:32,418] {taskinstance.py:1151} INFO - Marking task as UP_FOR_RETRY
[2020-06-01 20:18:41,862] {logging_mixin.py:112} INFO - [2020-06-01 20:18:41,861] {local_task_job.py:103} INFO - Task exited with return code 1
